### PR TITLE
add "What's next" on LiveTV Fullscreeninfo

### PR DESCRIPTION
**Pull request by walterheisenberg2**
info on LiveTV:
"up" switches between "FullPlot" and small "PlotOutline"
"left/right" switches between "actual" and "Next"-Plot-info

### CODE REVIEW

File: 1080i/DialogFullScreenInfo.xml [hunk 1]
```diff
@@ -336,100 +351,165 @@
...
+				<visible>IsEmpty(VideoPlayer.PlotOutline) | String.IsEqual(VideoPlayer.PlotOutline,N/A) | Skin.HasSetting(Enable.LiveTVFullPlot)</visible>
+			</control>
+			<!-- NOW playing in channel -->
+			<control type="grouplist">
+				<top>50</top>
+				<left>57</left>
+				<height>650</height>
+				<width>1805</width>
+				<itemgap>10</itemgap>
+				<height>750</height>
```
Comment (BigNoid) [inline 1080i/DialogFullScreenInfo.xml]: double height tag

File: 1080i/DialogFullScreenInfo.xml [hunk 2]
```diff
@@ -336,100 +351,165 @@
...
-					<visible>false</visible>
+			</control>
+			<!-- NEXT playing on channel -->
+			<control type="grouplist">
+				<top>50</top>
+				<left>57</left>
+				<height>650</height>
+				<width>1805</width>
+				<itemgap>10</itemgap>
+				<height>750</height>
```
Comment (BigNoid) [inline 1080i/DialogFullScreenInfo.xml]: double height tag

**Comment by walterheisenberg2** [comment]
I'll correct that later
Thx for the hint

FYI: Other PR that you merged:
you don't see any difference between Plot and PlotOutline in LiveTV-democlient
PlotOutline in "real" TV has e.g. the episodename and the year

**Comment by walterheisenberg** [comment]
Please have a look again.

**Comment by BigNoid** [comment]
Thx!

**Pull request merged**